content/cloud/labels.md

```diff
@@ -0,0 +1,65 @@
+
+> At this time, only NoOps SQL DB supports `labels`. However, in the future other Fermyon Cloud resources such as Key Value Store will support `labels` as well. 
+
+## Benefits of Labels
+
+Benefits of the Fermyon Clould `labels` include:
+
+* **Easy Sharing**: Share your Fermyon Cloud resource across applications effortlessly.
+* **Resource Creation Control**: You decide when to create and delete Fermyon Cloud resources.
+* **Flexible Lifecycle Management**: Manage Fermyon Cloud resources independently of your Spin application.
+* **Seamless Cloud Integration**: Cloud experience smoothly integrates with local Spin development, no runtime configuration changes required. 
+* **Component Model Alignment**: Change the data resource your application points to without rebuilding it.
+
+## How To Create A Label
+
+Labels can be created in two different manners, either within the component manifest in `spin.toml` or via `spin cloud` plugin via the `spin cloud link` command. Both approaches are equally valid. The path you choose most likely depends on whether your Spin application needs to be linked to the resource immediately at deployment time. If you know the exact topology of your workload, you'll likely include the label in the component manifest in your `spin.toml` at deployment time.
+
+On the other hand, if you're familiarizing yourself with Fermyon Cloud and interacting with these resources for the first time you may find yourself using `spin cloud link` instead. The `spin cloud link` command allows you to create a label once you already have a deployed Spin application and storage resource you'd like to connect to. 
+
+![TODO - this is a rough PM diagram, needs design polish](../../static/image/labels-diagram.png)
+
+## How to View Existing Labels
+
+To see your existing labels, you will need to use the `spin cloud` plugin along with the subcommand of the resource you've linked to your Spin application via a label. For example, if you'd like to see the labels you created for your NoOps SQL DB resources, you'd run the following command:
+
+```bash
+$ spin cloud sqlite list
++------+---------+--------------+
+| App  | Label   | Database     |
++===============================+
+| todo | default | amiable-kiwi |
+|------+---------+--------------|
+| api  | hello   | amiable-kiwi |
```

Comment:
"hello" and "amiable-kiwi" are unhelpful - consider names that offer some sense of why this sharing is happening.